ncm-ceph/src/main/pan/components/ceph/ceph-user.pan

```diff
@@ -6,14 +6,25 @@
 
 unique template components/${project.artifactId}/ceph-user;
 
-include { 'components/accounts/config' };
+include 'components/accounts/config';
+
+# do not change uids of existing cluster
+variable CEPH_OLD_UID ?= true; # set to false for new clusters, especially infernalis and above
+variable CEPH_USER_ID = {
+    if (CEPH_OLD_UID) {
+        deprecated(0, 'ceph user should get a new uid');
```

Comment:
add a reference to `(set final CEPH_OLD_UID = false)` somehow in the derecation message